### PR TITLE
WIP: add version parser

### PR DESCRIPTION
**Pull request by gdams**
Changes the directory of the extracted tarball to be `jdk-8.0.202+08.1` or `jre-8.0.202+08.1`

@smlambert please confirm whether any changes will be required to the test scripts with this new directory name?

closes: https://github.com/AdoptOpenJDK/openjdk-build/issues/617

### CODE REVIEW

**Comment by karianna** [comment]
I thought we already had this logic in the API and/or Website repo...

**Comment by gdams** [comment]
@johnoliver wrote this is groovy to write the metadata but this wasn't useful because we needed to know the metadata at build stage in order to name the directory accordingly.

File: version-parser/version-parser.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+try:
+    isinstance(int(split_version[0]), (int, long))
+except ValueError:
+    split_version = version.split('+')
+
+if int(split_version[0]) > 1:
+    # detected openjdk 9 or above
+    major = int(split_version[0]) # 11
+    try:
+        minor = int(split_version[1]) # 0
```
Comment (johnoliver) [inline version-parser/version-parser.py]: does this work for `11+28`?

Comment (gdams) [reply version-parser/version-parser.py]: tested here: https://github.com/AdoptOpenJDK/openjdk-build/blob/21cd7cf4162f444fa9ceb95c87f3f71870dd8bca/version-parser/test-version-parser.sh#L68-L70

**Comment by karianna** [comment]
As discussed on the call - this may not be the optimal approach, will see what you and John come up with on take 2 :-)

**Pull request closed without merging**